style.css

```diff
@@ -13,16 +13,87 @@ ul {
   margin: 0;
 }
 
-.body-m1 {
-  width: 400px;
-  background-color: #f4f5f7;
+.hamburger {
+  display: block;
+}
+
+.desktop-menu {
+  display: none;
 }
 
 .container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 36px 0 24px;
+  background: #fff;
+  height: 48px;
+  position: fixed;
+  width: 100vw;
+}
+
+/* .body-m1 {
+  width: 400px;
+  background-color: #f4f5f7;
+} */
+
+/* .container {
   display: flex;
   justify-content: space-between;
   align-items: center;
   background-color: white;
+} */
```

Comment:
- [ ] Kindly remove this piece of old commented code.  That way, the repo will be cleaner 💪

Reply:
@Meltrust Thank you for the review.